target/linux/ath79/generic/base-files/lib/upgrade/openmesh.sh

```diff
@@ -68,7 +68,8 @@ platform_do_upgrade_openmesh()
 	# boot anymore to Linux until it was reflashed with ap51-flash.
 	local next_boot_part="1"
 	case "$(board_name)" in
-	openmesh,om2p-v4)
+	openmesh,om2p-v4|\
+	openmesh,om2p-hs-v4)
```

Comment:
I wonder whether we really need another case here.

One could just make that another argument and then e.g. call `platform_do_upgrade_openmesh "$1" 3` in upgrade.sh

But please take this as an open thought, I'm not sure whether this applies to common behavior with these scripts and whether it's a good idea at all eventually ...

Reply:
will be dropped completely